.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,25 @@
+// File only needed for VSCode users to have proper Docker based interpreters
+{
+    "name": "accelerate_dev_environment",
+    "build": {
+        // ACTION NEEDED: comment/uncomment the relevant line depending on whether you are in a CPU/GPU environment
+        // "dockerfile": "../docker/accelerate-cpu/Dockerfile"
```

Comment:
I think we should enable CPU by default and have the user work on the container to activate GPU.

Reply:
Sure, I was torn on this, so am happy to go the other way.